apps/analysis/admin.py

```diff
@@ -32,3 +34,13 @@ class AnalyticalStatementAdmin(admin.ModelAdmin):
 @admin.register(DiscardedEntry)
 class DiscardedEntryAdmin(admin.ModelAdmin):
     pass
+
+
+@admin.register(AnalyticalStatementGeoEntry)
+class AnalyticalStatementGeoEntryAdmin(admin.ModelAdmin):
+    pass
```

Comment:
Let's add autocomplete fields

Reply:
just use for the test 
i will remove this code from admin.oy